functions/visualizations.py

```diff
@@ -3,8 +3,15 @@
 from io import BytesIO
 from typing import Optional, Union
 
+# clustergrammer2 via sklearn uses np.float which is deprecated as of numpy==1.20
+import warnings
+
+warnings.filterwarnings(
+    action="ignore", category=DeprecationWarning, module="scikit-learn"
+)
 import pandas as pd
-from clustergrammer import Network as CGNetwork
+
+from clustergrammer2 import Network as CGNetwork
```

Comment:
Have you confirmed that this works with the current frontend code? We don't use clustergrammer2 on the frontend.

Reply:
not yet oops

Reply:
from comparing [CG2's initialization](https://github.com/ismms-himc/clustergrammer2/blob/d9f28fd8fbc861ad0833420f350c59a21d3f012e/clustergrammer2/clustergrammer_fun/initialize_net.py#L73) to [CG's](https://github.com/MaayanLab/clustergrammer-py/blob/ea4c517e860e4afd1063164d98192b87b845fa39/clustergrammer/initialize_net.py#L49), `net.viz` has the structure

Reply:
![Image Pasted at 2021-5-4 17-44](https://user-images.githubusercontent.com/7273167/117075938-a1715b80-ad03-11eb-85e1-6c73fbf2e59b.png)
a quick test shows that the results match the shape of that already in use on prod